scipy/io/mmio.py

```diff
@@ -505,7 +532,7 @@ def _parse_body(self, stream):
         return a
 
     #---------------------------------------------------------------------------
-    def _write(self, stream, a, comment='', field=None, precision=None):
+    def _write(self, stream, a, comment='', field=None, precision=None, symmetry=None):
```

Comment:
Yes, that is right. I think, as well, that it makes most sense to use `symmetry=None` as default.